p2p/gossip/protocol.go

```diff
@@ -149,7 +150,7 @@ func (prot *Protocol) propagateMessage(msg []byte, h hash) {
 	prot.peersMutex.RLock()
 	for p := range prot.peers {
 		peer := prot.peers[p]
-		prot.Debug("sending message to peer %v, hash %d", peer.pubKey, h)
+		prot.Info("sending message to peer %v, hash %d", peer.pubKey, h)
```

Comment:
debug ? we have prints about this also in `swarm.SendMessage`

Reply:
set it back to debug